### PR TITLE
version lock mariadb until chef15 migration

### PR DESCRIPTION
**Pull request by Vancelot11**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 depends          'firewall'
 depends          'git'
+depends          'mariadb', '~> 3.2.0' # remove after chef15 upgrade
```
Comment (ramereth) [inline metadata.rb:14]: This is a better way to do it I think.
```suggestion
# TODO: Remove after chef15 upgrade
depends          'mariadb', '< 4.0.0'
```

**Comment by ramereth** [comment]
[version lock mariadb until chef15 migration](https://trello.com/c/1OVGgkVs/1720-version-lock-mariadb-until-chef15-migration)

**Comment by Vancelot11** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.3.2.  Have a nice day!

File: metadata.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 depends          'firewall'
 depends          'git'
+# TODO: Remove after chef15 upgrade
+depends          'mariadb', '~> 3.2.0'
```
Comment (ramereth) [inline metadata.rb:15]: Shoot, I missed that this should have been updated to use ``< 4.0.0``

Comment (ramereth) [reply metadata.rb:15]: I'll fix this in another PR.

**Pull request merged**